test/built-ins/Object/getOwnPropertyDescriptor/length.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2012 Ecma International. All rights reserved.
```

Comment:
Since this is an entirely new file, containing an entirely different mechanism for testing, I'd say it cannot possibly be "Copyright (C) 2012 Ecma International."